### PR TITLE
{vis}[GCC/10.3.0] GIMP v2.10.24

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~#13196~ (Graphviz), ~~#13202~~ (GTK+), ~#13203~ (GnuTLS), ~#13204~ (gexiv2), Boost (~#13054~)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/p/poppler/poppler-21.06.1-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+dependencies = [
+    ('freetype', '2.10.4'),
+    ('fontconfig', '2.13.93'),
+    ('libjpeg-turbo', '2.0.6'),
+    ('libpng', '1.6.37'),
+    ('NSS', '3.65'),
+    ('LibTIFF', '4.2.0'),
+    ('Qt5', '5.15.2'),
+    #('Boost', '1.76.0'),
```
Comment (boegel) [inline easybuild/easyconfigs/p/poppler/poppler-21.06.1-GCCcore-10.3.0.eb:25]: This should be fixed, but that requires #13054 ...

Comment (verdurin) [reply easybuild/easyconfigs/p/poppler/poppler-21.06.1-GCCcore-10.3.0.eb:25]: @boegel seems like we're still stuck on https://github.com/easybuilders/easybuild-easyblocks/pull/2456

Comment (SebastianAchilles) [reply easybuild/easyconfigs/p/poppler/poppler-21.06.1-GCCcore-10.3.0.eb:25]: Boost was merged in https://github.com/easybuilders/easybuild-easyconfigs/pull/12977
```suggestion
    ('Boost', '1.76.0'),
```

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 7 out of 9 (8 easyconfigs in total)
node3100.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/709dc6838b0ac882cec39525ead724d7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
node3100.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/7a869edbe4b3ae8cc00f3df6b73b89be for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/p/PyTorch/PyTorch-1.8.1-fosscuda-2020b.eb [hunk 2]
```diff
@@ -116,6 +116,8 @@ excluded_tests = {
         # Potentially problematic save/load issue with test_lstm on only some machines. Tell users to verify save&load!
         # https://github.com/pytorch/pytorch/issues/43209
         'test_quantization',
+        # hangs?
+        'test_profiler',
```
Comment (branfosj) [inline easybuild/easyconfigs/p/PyTorch/PyTorch-1.8.1-fosscuda-2020b.eb:120]: Accidental addition?

File: easybuild/easyconfigs/p/poppler/poppler-21.06.1-GCC-10.3.0.eb [hunk 3]
```diff
@@ -11,6 +11,7 @@ toolchain = {'name': 'GCC', 'version': '10.3.0'}
 builddependencies = [
     ('CMake', '3.20.1'),
     ('pkg-config', '0.29.2'),
+    ('Python', '3.9.5'),
```
Comment (branfosj) [inline easybuild/easyconfigs/p/poppler/poppler-21.06.1-GCC-10.3.0.eb:14]: Can/should we use the `-bare` Python here?

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/p/PyTorch/PyTorch-1.8.1-fosscuda-2020b.eb (hunk 2 above)
Comment (boegel) [reply easybuild/easyconfigs/p/PyTorch/PyTorch-1.8.1-fosscuda-2020b.eb]: indeed, fixed with force push

**Review by branfosj — APPROVED** [review]
lgtm

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/227d41b7f9591e32036c770e96ca0c05 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**